src/main/java/com/microsoft/cse/helium/app/services/keyvault/KeyVaultService.java

```diff
@@ -9,14 +9,14 @@
 import com.microsoft.azure.keyvault.models.KeyBundle;
 import com.microsoft.azure.keyvault.models.SecretBundle;
 import com.microsoft.azure.keyvault.models.SecretItem;
+import com.microsoft.cse.helium.app.Constants;
```

Comment:
are  we removing the .microsoft from the namespace?

Reply:
Keeping it inline with other projects, we probably should. I added an item to the triage #149